src/main/java/ddingdong/ddingdongBE/domain/form/service/FacadeCentralFormServiceImpl.java

```diff
@@ -68,9 +69,12 @@ public void createForm(CreateFormCommand createFormCommand) {
 
         Form form = createFormCommand.toEntity(club);
         Form savedForm = formService.create(form);
-
-        List<FormField> formFields = toCreateFormFields(savedForm,
-                createFormCommand.formFieldCommands());
+        List<FormField> formFields;
+        if (savedForm.getSections().size() > 1) {
+            formFields = toCreateFormFieldsForMultipleSections(savedForm, createFormCommand.formFieldCommands());
+        } else {
+            formFields = toCreateFormFieldsForSingleSection(savedForm, createFormCommand.formFieldCommands());
+        }
```

Comment:
p2)
formFields를 초기화 하는 부분을 private으로 분리하면 가독성이 더 좋아질 것 같네요
```suggestion
        List<FormField> formFields = buildFormFieldBySection(savedForm, createFormCommand.formFieldCommands())
```

Reply:
고민했던 부분인데 감사합니다! 수정하였습니다.